app/views/generic_files/_generic_file.html.erb

```diff
@@ -1,6 +1,7 @@
 <li>
   <h2>
     <%= link_to generic_file.title_or_label, sufia.generic_file_path(generic_file) %>
+    <small>[<%= link_to 'Download', download_path(generic_file), target: '_blank', rel: 'noopener noreferrer' %>]</small>
```

Comment:
Should we have the same logic here - to not have the download link if there's no file attached?